### PR TITLE
Process return types

### PR DESCRIPTION
**Pull request by fvovan**
#2 
generate functions' return types

I added type annotations to some of the classes to emphasize it made for v7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-  - 5.6
+  - 7.2
```
Comment (dlondero) [inline .travis.yml:4]: Maybe add `7.3` and `7.4` directly?

**Review by dlondero — APPROVED** [review]
LGTM
Only comment to add versions to the PHP matrix on Travis.

File: .travis.yml (hunk 1 above)
Comment (fvovan) [reply .travis.yml:4]: it turned out we had to upgrade phpunit in order to comply with `7.3` and `7.4` deprecations

Comment (dlondero) [reply .travis.yml:4]: Yep, there was https://github.com/cargomedia/codegenerator/issues/1 for that. Two birds with one stone!

**Review by dlondero — APPROVED** [review]

**Pull request merged**